code.py

```diff
@@ -0,0 +1,11 @@
+import json
+
+
+def lambda_handler(event, context):
+    if "body" not in event:
+        return "no body found"
```

Comment:
> [!CAUTION]
> The function returns a string instead of a proper API Gateway response format. Lambda functions triggered by API Gateway should return a dictionary with statusCode, headers, and body.

> [!TIP]
> Return a properly formatted API Gateway response with statusCode, headers and body properties.

```python
return {
    'statusCode': 400,
    'headers': {'Content-Type': 'application/json'},
    'body': json.dumps({'error': 'no body found'})
}
```